travel_platform/api/izi_travel/izi_travel_client.py

```diff
@@ -1,3 +1,4 @@
+from multiprocessing.pool import ThreadPool
```

Comment:
Is this import using somewhere in this file?
If yes, please provide line number, because I can't find.Thanks!